jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/asserts/IconAssert.java

```diff
@@ -3,44 +3,54 @@
 import com.epam.jdi.light.asserts.generic.UIAssert;
 import com.epam.jdi.light.common.JDIAction;
 import com.epam.jdi.light.vuetify.elements.common.Icon;
-import com.epam.jdi.light.vuetify.interfaces.asserts.AlignmentAssert;
-import com.epam.jdi.light.vuetify.interfaces.asserts.CursorAssert;
-import com.epam.jdi.light.vuetify.interfaces.asserts.MeasurementAssert;
-import com.epam.jdi.light.vuetify.interfaces.asserts.ThemeAssert;
+import com.jdiai.tools.Timer;
 import org.hamcrest.Matchers;
 
 import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
 
-public class IconAssert extends UIAssert<IconAssert, Icon> implements CursorAssert<IconAssert, Icon>,
-		AlignmentAssert<IconAssert, Icon>, ThemeAssert<IconAssert, Icon>, MeasurementAssert<IconAssert, Icon> {
+public class IconAssert extends UIAssert<IconAssert, Icon> {
+
+	@JDIAction("Assert that '{name}' is displayed")
+	public IconAssert displayed() {
+		Timer.waitCondition(element()::isDisplayed);
+		jdiAssert(element().isDisplayed(), Matchers.is(true));
+		return this;
+	}
 
 	@JDIAction("Assert that '{name}' is clickable")
 	public IconAssert clickable() {
-		jdiAssert(element().core().isClickable(), Matchers.is(true), "Element is not clickable");
+		jdiAssert(element().core().isClickable() ? "is clickable" : "is not clickable",
+			Matchers.is("is clickable"));
 		return this;
 	}
 
 	@JDIAction("Assert that '{name}' has expected type")
 	public IconAssert type(String iconType) {
-		String actualIconType = element().hasType();
-		jdiAssert(actualIconType, Matchers.is(iconType), String.format("Element's actual icon type '%s' is not equal " +
-				"to expected '%s'", actualIconType, iconType));
+		jdiAssert(element().hasType(), Matchers.is(iconType));
+		return this;
+	}
+
+	@JDIAction("Assert that '{name}' has expected height")
+	public IconAssert height(Integer height) {
+		jdiAssert(element().hasHeight(), Matchers.is(String.format("%spx", height)));
+		return this;
+	}
+
+	@JDIAction("Assert that '{name}' has expected width")
+	public IconAssert width(Integer width) {
+		jdiAssert(element().hasWidth(), Matchers.is(String.format("%spx", width)));
 		return this;
 	}
 
 	@JDIAction("Assert that '{name}' has expected color")
 	public IconAssert color(String color) {
-		String actualColor = element().color();
-		jdiAssert(actualColor, Matchers.is(color), String.format("Actual element's color '%s' is not equal to " +
-				"expected '%s'", actualColor, color));
+		jdiAssert(element().hasColor(), Matchers.is(color));
```

Comment:
Please, look through interfaces created for Vuetify. Team agreed to use approach with interfaces.